OIIO/WriteOIIO.cpp

```diff
@@ -816,19 +825,21 @@ WriteOIIOPlugin::refreshParamsVisibility(const string& filename)
                            strcmp(output->format_name(), "avif") == 0 ||
                            output->supports("quality"));
         bool hasDWA = false;
+        bool hasZIP = false;
         bool isEXR = strcmp(output->format_name(), "openexr") == 0;
         bool isTIFF = strcmp(output->format_name(), "tiff") == 0;
         if (isEXR || isTIFF) {
             int compression_i;
             _compression->getValue(compression_i);
             EParamCompression compression = (EParamCompression)compression_i;
-            hasQuality = (compression == eParamCompressionJPEG ||
-                          compression == eParamCompressionZip);
+            hasQuality = (compression == eParamCompressionJPEG);
             if (isEXR) {
                 hasDWA = (compression == eParamCompressionDWAa) || (compression == eParamCompressionDWAb);
+                hasZIP = (compression == eParamCompressionZip) || (compression == eParamCompressionZips);
```

Comment:
It sounds like this line should be moved out of `if(isEXR)`, to be enable zipCompressionLevel for TIFF.
Can you please check that @360highdynamicrangehdr ?